client/components/App.jsx

```diff
@@ -40,7 +40,7 @@ class App extends React.Component {
     const restaurantId = window.location.href.slice(36);
     $.ajax({
       type: 'GET',
-      url: 'api/restaurants/'+restaurantId,
+      url: 'localhost:3005/api/restaurants/'+restaurantId,
```

Comment:
by removing `localhost,` you have fewer problems with express. It uses the server that served the static files when you do not include it.

Reply:
Agree! I'll change that.